CONTRIBUTORS.md

```diff
@@ -63,7 +63,7 @@ projects:
 ## Sponsors
 
 The following organisations have financially supported the continued
-developent and maintainence of Sonic Pi. We give our appreciation and
+developent and maintenance of Sonic Pi. We give our appreciation and
```

Comment:
it seems there is another typo in the same sentence:

developent => development?

Reply:
Good catch! Happy to merge a PR fixing this :-)

Reply:
sure: https://github.com/sonic-pi-net/sonic-pi/pull/3194